README.md

```diff
@@ -82,11 +82,28 @@ More about development and contributing practices can be found in [`CONTRIBUTING
 
 ------
 
-## Compatibility with Cluster API and Kubernetes Versions
+## Compatibility with Cluster API
 
 - BYOH is currently compatible wth Cluster API v1beta1 (v1.0)
-- BYOH installer support is present and tested only for Kubernetes version `v1.22.3`. You may however use it to provision clusters of a different version by manually installing the Kubernetes components (use the `--skip-installation` flag when starting the agent)
 
+## Supported OS and Kubernetes versions
+|                                               | Kubernetes v1.21.* | Kubernetes v1.22.* | Kubernetes v1.23.* |
+| ----------------------------------------------| :----------------: | :----------------: | :----------------: |
+| byoh-bundle-ubuntu_20.04.1_x86-64_k8s:v1.21.* |        ✓           |        x           |        x           |
+| byoh-bundle-ubuntu_20.04.1_x86-64_k8s:v1.22.* |        x           |        ✓           |        x           |
+| byoh-bundle-ubuntu_20.04.1_x86-64_k8s:v1.23.* |        x           |        x           |        ✓           | 
+
+
+|                                               | Ubuntu_20.04.*_x86-64 |
+| ----------------------------------------------| :-------------------: |
+| byoh-bundle-ubuntu_20.04.1_x86-64_k8s:v1.21.* |          ✓            |
+| byoh-bundle-ubuntu_20.04.1_x86-64_k8s:v1.22.* |          ✓            |
+| byoh-bundle-ubuntu_20.04.1_x86-64_k8s:v1.23.* |          ✓            |
+
+
+**NOTE:**  The '*' in OS means that all Ubuntu 20.04 patches will be handled by this BYOH bundle.
+
+**NOTE:**  The '*' in the K8S Version means that the k8s minor release is supported but it may happen that a byoh bundle for a specific patch may not exist n the OCI registry,
```

Comment:
```suggestion
**NOTE:**  The '*' in the K8S Version means that the k8s minor release is supported but it may happen that a byoh bundle for a specific patch may not exist in the OCI registry,
```